src/d3d12/d3d12_model_pool.hpp

```diff
@@ -32,6 +32,51 @@ namespace wr
 			std::size_t m_index_staging_buffer_size;
 			std::size_t m_index_count;
 			void* m_index_memory_block;
+
+			bool data_changed;
+		};
+
+		enum CommandType
+		{
+			STAGE,
+			COPY,
+			READ,
+			TRANSITION,
+		};
+
+		struct Command
```

Comment:
Unused?

Reply:
Functions as a common ancestor for all different commands, so they can all fit into a single queue.